crate/operator/webhooks.py

```diff
@@ -275,5 +281,31 @@ async def send_upgrade_notification(
             logger=logger,
         )
 
+    async def send_delay_notification(
```

Comment:
I just realised that `send_delay_notification`, `send_scale_notification` and `send_upgrade_notification` are only used in tests. I wonder if we could use the generic `send_notification` method everywhere and get rid of these altogether?..

Reply:
Yeah I wondered the same, but since for the other operations we had these specific functions I added the same for `delay`. I tested it with the `send_notification` and it worked too, might be worth to remove these "testing" only functions and just use `send_notification` for all.

Reply:
That might be a good idea, up to you if you want to do it as part of this PR though!